Sources/App/routes.swift

```diff
@@ -3,7 +3,7 @@ import Vapor
 
 func routes(_ app: Application) throws {
     app.get { req in
-        render(page: homePage())
+        homePage()
```

Comment:
The `ResponseEncodable` extension on `HTML` above allows us to return `HTML` straight from the route handler and this is the entry point for `/`.